### DISCUSSION TITLE
Is there any plan to add date picker component ?

### DISCUSSION THREAD

**Discussion by ProPhycient5** [saadeghi/daisyui#1441] [Ideas / feature requests]
I think it's high time to have a Date picker dynamic component. Is it coming soon ?

**Comment by half-metal** [comment]
Having a daisyUI component for date picker would be great, very common for a lot of sites. Right now there's just standard looking ones, like this https://github.com/beyonk-adventures/svelte-datepicker. I was recently trying to figure out how to get a calendar that allows multiple dates/ranges and looks intuitive, but would be good to figure that javascript out with a daisyUI component.

**Comment by saadeghi** [comment]
Similar discussion here:
- https://github.com/saadeghi/daisyui/discussions/633
- https://github.com/saadeghi/daisyui/discussions/1004
